src/components/Button/Button.spec.tsx

```diff
@@ -30,15 +30,6 @@ describe('Button', () => {
     expect(await driver.isMobile()).toEqual(true);
   });
 
-  it('should expose focus method using ref', async () => {
```

Comment:
Why remove this test?
I think it's better to leave this, since this test was created to ensure this doesn't break.